_content/references.md

```diff
@@ -1,5 +1,170 @@
 ## References
 
+### Word list
+
+This section lists commonly used words and phrases in Agora technical documentation. For the usage of other words, refer to the [word list](https://developers.google.com/style/word-list) in Google developer documentation style guide.
+
+#### Agora-related naming conventions
+
+When referring to an Agora SDK, use "the". For example, "the Agora Native SDK" or "the Agora Recording SDK", and not "Agora Signaling SDK". 
+
+Do not use "the" for proper nouns such as "Agora Video Call" or "Agora Recording". Do not use "the Agora Cloud Recording".
+
+| **Full Name**                                | **Short Form**                      | **Description/Notes**                                        |
+| -------------------------------------------- | ----------------------------------- | ------------------------------------------------------------ |
+| Agora Analytics                              | Agora Analytics                     | A tool that tracks and analyzes the usage and quality of calls. |
+| Agora Cloud Recording                        | Cloud Recording                     | A component provided by Agora to record and save voice calls, video calls, and interactive broadcasts on your cloud storage. |
+| Agora Interactive Live Streaming Premium     | Interactive Live Streaming Premium  | Live streaming with ultra-low latency and strong interaction. |
+| Agora Interactive Live Streaming Standard    | Interactive Live Streaming Standard | Live streaming with low latency and light interaction.       |
+| Agora Video Call                             | Video Call                          | Implements high-quality one-to-one or group video calls.     |
+| Agora Voice Call                             | Voice Call                          | Implements crystal-clear one-to-one or group voice calls.    |
+| Agora Real-time Messaging                    | Agora Real-time Messaging           | Provides global messaging cloud service with high reliability, low latency, and high concurrency. |
+| One-to-one Classroom                         | One-to-one Classroom                | An online teacher gives an exclusive lesson to only one student. |
```

Comment:
A teacher gives an online lesson to only one student.
To be consistent with the other two use cases